code/__DEFINES/mobs.dm

```diff
@@ -71,6 +71,12 @@
 #define HUMAN_MAX_OXYLOSS 3
 #define HUMAN_CRIT_MAX_OXYLOSS (SSMOBS_DT/3)
 
+/// Base amonut for the maximum amount of delimbs a human can suffer from an explosion
+#define HUMAN_EXP_MAX_DELIMB 1.33
```

Comment:
These defines are not very useful or clear given severity is hidden behind a define. I think it would be clearer to just have as switch in the code.